src/js/components/ShareButton.jsx

```diff
@@ -11,48 +11,42 @@ import { encURI } from '../base/utils/miscutils';
  * description
  */
 class ShareButton extends React.Component {
```

Comment:
This looks good -- I love the popout tray of icons.

Why does it use `class extends React.Component`?   
It should work fine as a plain jsx function, with useState() for tray open/closed.